src/module/actor/character/document.ts

```diff
@@ -1646,11 +1646,6 @@ class CharacterPF2e extends CreaturePF2e {
             auxiliaryActions,
         });
 
-        // Define these as getters so that Foundry's TokenDocument#getBarAttribute method doesn't recurse infinitely
-        Object.defineProperty(action, "origin", {
-            get: () => this.items.get(weapon.id),
-        });
```

Comment:
Can you confirm that repeating ammo display still works after this?

Reply:
Then sure. This bit of code was added so that the number of repeating ammo would show in the strikes tab, but if that's no longer needed we can kill it.